### PR TITLE
Adding support for Interlay, Tinkernet, and update to the HydraDX configuration

### PR DESCRIPTION
**Pull request by x676f64**
I've added support for Interlay, Tinkernet, and included balances for all defined assets. I've also extended HydraDX's config to include all assets there.

### CODE REVIEW

File: configs/hydradx.yml [hunk 1]
```diff
@@ -1,22 +1,99 @@
 endpoint: wss://hydradx-rpc.dwellir.com
 mock-signature-host: true
 block: ${env.HYDRADX_BLOCK_NUMBER}
-db: ./db.sqlite
+db: ./hydradx.db.sqlite
 # wasm-override: hydradx_runtime.compact.compressed.wasm
 
 import-storage:
-  System:
```
Comment (ermalkaleci) [inline configs/hydradx.yml:8]: no need for system::account?

Comment (arrudagates) [reply configs/hydradx.yml:8]: Interlay uses Tokens for both native and external assets.

Comment (x676f64) [reply configs/hydradx.yml]: Thanks, this is needed. Added it back.

**Comment by ermalkaleci** [comment]
@x676f64 let me know when it's ready to merge

**Comment by x676f64** [comment]
> 



> @x676f64 let me know when it's ready to merge

Should be good to merge now

**Pull request merged**